Libraries/LLM/Evaluate.swift

```diff
@@ -179,7 +179,7 @@ public struct TokenIterator: Sequence, IteratorProtocol {
         // prepare the prompt in chunks if larger than the prefill size
         while y.size > parameters.prefillStepSize {
             _ = model(
-                y[..<parameters.prefillStepSize, .newAxis], cache: cache.isEmpty ? nil : cache)
+                y[.newAxis, ..<parameters.prefillStepSize], cache: cache.isEmpty ? nil : cache)
```

Comment:
This code from python was translated incorrectly:

```python
        model(y[:prefill_step_size][None], cache=cache)
```

should be:

```swift
y[.newAxis, ..<parameters.prefillStepSize]
```

So this only hits the preload case (> 512 tokens) and curiously it seems to work fine with smaller preloads -- I was testing it with 32 token parameters.prefillStepSize and didn't encounter this.

Anyway, this was a bad problem because the shape of the input to attention was:

python: (1, 512, 3072)
swift: [512, 1, 3072]